Makefile

```diff
@@ -1,5 +1,34 @@
-run:
-	python manage.py runserver
+.DEFAULT_GOAL := help
+
+help:
+    @grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+run: ## Run the django server with runserver_plus
+    python manage.py runserver_plus
```

Comment:
for now we can keep using the normal runserver, `runserver_plus` needs some configuration
https://django-extensions.readthedocs.io/en/latest/runserver_plus.html